pages/about/code-of-conduct.vue

```diff
@@ -241,23 +241,26 @@ export default {
 
 <style scoped>
 .paragraph-title {
-    @apply mt-8 mb-8 font-serif font-bold text-center text-left text-pink-700;
-    font-size: 24px;
+    @apply mt-8 mb-8 font-serif font-bold text-center text-left text-pink-700 text-2xl;
 }
 @media (min-width: 768px) {
     .paragraph-title {
-        font-size: 28px;
+        @apply text-3xl;
     }
 }
 .img_text {
-    @apply absolute mt-24 whitespace-nowrap text-center;
-    font-size: 0.8rem;
+    @apply absolute mt-24 whitespace-nowrap text-center text-sm;
 }
 @media (max-width: 640px) {
     .img_text {
         @apply mt-20;
     }
 }
+@media (max-width: 500px) {
+    .img_text {
+        @apply text-xs whitespace-normal;
+    }
+}
```

Comment:
may i ask why 500px ?
is there any cases you have to handle with 500px?
the below code is tailwind rule on responsive screen
```
screens: {
            xs: '360px',
            sm: '640px',
            md: '768px',
            lg: '1194px',
            xl: '1440px',
            '2xl': '1600px',
        },
```